src/runtime/config/configuration-clh-snp.toml.in

```diff
@@ -137,7 +137,7 @@ default_maxmemory = @DEFMAXMEMSZ@
 #   - virtio-fs (default)
 #   - virtio-fs-nydus
 #   - none
-shared_fs = "@DEFSHAREDFS_CLH_SNP_VIRTIOFS@"
+shared_fs = "none"
```

Comment:
we should still leave the variables, to adapt the variables in our package_build.sh accordingly

Reply:
Separated virtio-blk kata pod-sandboxing into a separate config.